app/views/handlers/block.py

```diff
@@ -29,6 +29,7 @@ def __init__(
         self.form = None
         self.page_title = None
         self._return_to_summary = self._request_args.get("return_to_summary")
+        self.resume = self._request_args.get("resume")
```

Comment:
Good point. We should validate the query parameters and parse them into booleans.

Reply:
We should also consider appropriately casing the query params i.e. `true` rather than `True`.

Reply:
If we used to the inverse condition to control anything then I would agree (True do this, False do that) however we don't. We are basically using the presence of the param, at which point the value largely becomes redundant. Realistically we should just pass the param if the value means nothing, which I believe is still valid. One for tech session? should only take 5 minutes (more likely 20)

Reply:
> We should also consider appropriately casing the query params i.e. `true` rather than `True`.

I also prefer this ☝️ 

Query params has to have a value, if the params didn't equal anything it would evaluate false when using `self._request_args.get()`.

Reply:
Surely you just then do a "resume" in self._request_args